pyspeckit/spectrum/models/model.py

```diff
@@ -307,18 +319,6 @@ def f(p,fjac=None):
                 return [0,residuals]
         return f
 
-    def __call__(self, *args, **kwargs):
```

Comment:
The `__call__` method should remain; there is no reason to remove it.  Just remove the `multisingle` parsing part of it.